cocotb/share/makefiles/simulators/Makefile.aldec

```diff
@@ -144,36 +144,14 @@ ifeq ($(COVERAGE),1)
 endif
 endif
 
-ifeq ($(OS),Msys)
-
-# Windows allows the situation where the libstc++ used at link time as
-# specified by -L can be different to the one that is used at runtime which
-# comes from the first libstdc++ that it finds in the path. As such
-# we use the mingw lib used at build time and put this at the start of the path
-# before running
```

Comment:
What happened to this comment? Was it wrong?

Reply:
On all configurations, I tested it was not needed. I also would consider this a user issues with wrong `PATH` settings. In `conda` and `msys2` prompts the path to `gcc` is set at the beginning of `PATH` and there is no need to manipulate this. I do not know how this was in mingw. I would guess the same issue will be on any other platform.